<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiCreateTextFileDialog</class>
 <widget class="QDialog" name="GuiCreateTextFileDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Workgroups</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPlainTextEdit" name="mp_text"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="mp_rbSendCode">
       <property name="text">
        <string>Send as text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mp_rbSendFile">
       <property name="text">
        <string>Send as file</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="mp_lFileSuffix">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>File extension</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mp_leFileSuffix">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbSend">
       <property name="minimumSize">
        <size>
         <width>38</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>38</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Send</string>
       </property>
       <property name="icon">
        <iconset resource="../beebeep.qrc">
         <normaloff>:/images/send.png</normaloff>:/images/send.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mp_text</tabstop>
  <tabstop>mp_rbSendCode</tabstop>
  <tabstop>mp_rbSendFile</tabstop>
  <tabstop>mp_leFileSuffix</tabstop>
  <tabstop>mp_pbSend</tabstop>
 </tabstops>
 <resources>
  <include location="../beebeep.qrc"/>
 </resources>
 <connections/>
</ui>
